workers/ohsome_quality_analyst/indicators/currentness/indicator.py

```diff
@@ -16,6 +16,10 @@ class Currentness(BaseIndicator):
     Ratio of all contributions that have been edited since 2008 until the
     current day in relation with years without mapping activities in the same
     time range
+
+    Attributes:
+        threshold (int): Number of years it should be since more than
+        50% of the elements were last edited
```

Comment:
```suggestion
        threshold_class_5 (int): Number of years ...
        threshold_class_4 (int):  ...
        threshold_class_3 (int): ...
        threshold_class_2 (int): ...
```

Reply:
After seeing it like this I think we should stick to the convention and this means listing the real attributes: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Done